### PR TITLE
Ana/fix multi send to address

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

This is the real fix the bug my PR #3357 intended to address

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/transactions/message-view/MultiSendMessageDetails.vue [hunk 1]
```diff
@@ -11,7 +11,7 @@
       </template>
       <template v-else-if="sentFromSessionAddress">
         <span>To&nbsp;</span>
-        <Bech32 :address="transaction.value.to_address" />
+        <Bech32 :address="transaction.value.outputs[0].address" />
```
Comment (faboweb) [inline src/components/transactions/message-view/MultiSendMessageDetails.vue]: Hmm what happens to the other output addresses?

Comment (faboweb) [reply src/components/transactions/message-view/MultiSendMessageDetails.vue]: We can add this fix and discuss an improvement later. Want to open an issue?

Comment (Bitcoinera) [reply src/components/transactions/message-view/MultiSendMessageDetails.vue]: > Hmm what happens to the other output addresses?

There are no other addresses :woman_shrugging: It works like the `inputs`, which is already implemented this way in this same file.

But yes, we can add this now and then see later. It should be fixed though, I think.

Comment (faboweb) [reply src/components/transactions/message-view/MultiSendMessageDetails.vue]: Inputs and outputs are arrays as far as I see. Have you considered, that there could be multiple in and outputs in a multi send? Maybe the implementation before is already faulty.

Comment (Bitcoinera) [reply src/components/transactions/message-view/MultiSendMessageDetails.vue]: Yes, I finally realized what you meant... It is really worth investigating

**Pull request merged**